pkg/bytes/range.go

```diff
@@ -13,7 +13,7 @@ type Range struct {
 }
 
 func (r Range) String() string {
-	return fmt.Sprintf(`{"Offset":"0x%x", "Length":"0x%x"}`, r.Offset, r.Length)
+	return fmt.Sprintf(`{"Offset":"%d", "Length":"%d"}`, r.Offset, r.Length)
```

Comment:
If this is integers, they might be without quotes.

Reply:
Sorry, I did it for local debug